website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Release Notes - Podman Desktop 1.7
+description: Podman Desktop 1.7 has been released!
+slug: podman-desktop-release-1.7
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.7/renovations.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.7 Release! 🎉
+
+![Podman-desktop-1-7-hero](img/podman-desktop-release-1.7/renovations.png)
+
+This release introduces:
+
+- **Podman 4.9.0**: [Podman 4.9.0](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Extension API Improvements**: Another set of improvements to the extension API enabling more goodness for 🦭 Podman Desktop's extensions.
+- **Experimental Kubernetes UI**: 
+
+Podman Desktop 1.7 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Extension API improvements
+
+The 🦭 Podman Desktop extension API received many improvements, including:
+
+- Add webview in the UI [#5594](https://github.com/containers/podman-desktop/pull/5594)
+- Add webview API for extensions [#5592](https://github.com/containers/podman-desktop/pull/5592)
+- Allow extensions to list webviews [#5628](https://github.com/containers/podman-desktop/pull/5628)
+- Allow to create container without starting it [#5643](https://github.com/containers/podman-desktop/pull/5643)
+- Expose create/start Pod and replicatePodmanContainer [#5648](https://github.com/containers/podman-desktop/pull/5648)
+- Expose create/list/delete volumes for extensions [#5598](https://github.com/containers/podman-desktop/pull/5598)
+- Add getImageInspect to API [#5596](https://github.com/containers/podman-desktop/pull/5596)
+- Add new contribution points for icon of image [#5543](https://github.com/containers/podman-desktop/pull/5543)
+- Adding BuildOption [#5533](https://github.com/containers/podman-desktop/pull/5533)
+- Add platform parameter to image build method [#5501](https://github.com/containers/podman-desktop/pull/5501)
+- Expose build image method [#5500](https://github.com/containers/podman-desktop/pull/5500)
+- Adding navigation api [#5558](https://github.com/containers/podman-desktop/pull/5558)
+
+### Experimental Kubernetes UI
+
```

Comment:
"We've sealed up a new experimental Kubernetes UI to support additional Kubernetes objects. We'd love if you're willing to 'break the seal' and run in experimental mode - give us your feedback on this emerging, work-in-progress feature so once complete it'll go swimmingly for your use cases!"

Reply:
I love a good seal pun, but this is hitting max pun-per-sentence. 😂 I included most of this but spread it out across a few paragraphs, feel free to review again and suggest if you think it's not enough.